spec/system/consumer/split_checkout_tax_incl_spec.rb

```diff
@@ -137,11 +142,65 @@
           end
 
           # DB check
-          order_within_zone.reload
-          voucher_adjustment = order_within_zone.voucher_adjustments.first
+          assert_db_voucher_adjustment(-10.00, -1.15)
+        end
+
+        describe "moving between summary to summary via edit cart" do
+          let!(:voucher) do
+            create(:voucher, code: 'good_code', enterprise: distributor, amount: 15)
+          end
+
+          it "recalculate the tax component properly" do
+            visit checkout_step_path(:details)
+            proceed_to_payment
+
+            # add Voucher
+            fill_in "Enter voucher code", with: voucher.code
```

Comment:
I would spell it out.
```suggestion
            fill_in "Enter voucher code", with: "good_code"
```